### DISCUSSION TITLE
Inconsistent short description for Patient.date-of-arrival extension

### DISCUSSION THREAD

**Discussion by techintrorepo** [hl7au/au-fhir-core#48] [Consensus Discussions for R1]
In Patient profile, the description of ([extension:dateOfArrival](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-patient-definitions.html#diff_Patient.extension:dateOfArrival) says: 'Date of first arrival in Australia' and data type 'date' is aligned with the description. However in Patient profile the short description is : 'Year of Arrival'. Knowing 'date' datatype can support only year, is it encouraged to use 'Year' only?

**Comment by dt-r** [comment]
Hi @techintrorepo, if this is to feed into the AU Core Patient consensus discussion this comment should be replicated onto: [AU Core Patient Consensus Discussion #36](https://github.com/hl7au/au-fhir-core/discussions/36)

**Comment by dt-r** [comment]
Moved to https://github.com/hl7au/au-fhir-core/discussions/36
